v9: build are not available on GitHub (#2007)
PrestaShop/docs: basics/installation/_index.md

If you intend to work on PrestaShop itself, we suggest using Git to clone the source code of PrestaShop from the GitHub repository.
{{% /notice %}}

{{% notice warning %}}

> Starting with PrestaShop 9, releases versions are not build on GitHub and are not available on GitHub anymore. Read our [article](https://build.prestashop-project.org/news/2024/new-zip-distribution-channel/) on Build blog for more informations about it.

Reviewer: ```suggestion
Starting with PrestaShop 9, releases versions are not built on GitHub and are not available on GitHub anymore. Read our [article](https://build.prestashop-project.org/news/2024/new-zip-distribution-channel/) on Build blog for more informations about it.
```

> Starting with PrestaShop 9, releases versions are not built on GitHub and are not available on GitHub anymore. Read our [article](https://build.prestashop-project.org/news/2024/new-zip-distribution-channel/) on Build blog for more informations about it.

Reviewer 2: ```suggestion
Starting with PrestaShop 9, release versions are no longer built or available on GitHub. For more information, read our [article](https://build.prestashop-project.org/news/2024/new-zip-distribution-channel/) on the Build blog.
```
